iina/ThumbnailCache.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ThumbnailCache.swift
+//  iina
+//
+//  Created by lhc on 14/6/2017.
+//  Copyright © 2017 lhc. All rights reserved.
+//
+
+import Cocoa
+
+class ThumbnailCache {
+
+  static private let sizeofDouble = MemoryLayout<Double>.size
+  static private let sizeofInt64 = MemoryLayout<Int64>.size
```

Comment:
Technically `Int` could be `Int32`(although it's not possible in IINA) or `Int64`, so just to make sure make it's exactly 4 bytes.